metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -204,20 +205,22 @@ object ServerCommands {
     "[string], where the string is a stacktrace."
   )
 
-  val ShowTasty = new ParametrizedCommand[TextDocumentPositionParams](
-    "show-tasty",
-    "Show TASTy",
-    """|If the file is a Scala 3 source, this command will try to find the relevant tasty file for it, 
-       |read it and display it in a human readable format. If the argument already points to a TASTy file, 
-       |it will be read directly.""".stripMargin,
-    """|
-       |Object with `document` and `position`, where the document is a path to a Scala 3 source or a tasty file.
+  final case class ChooseClassRequest(
+      textDocument: TextDocumentIdentifier,
+      includeInnerClasses: Boolean
+  )
+  val ChooseClass = new ParametrizedCommand[ChooseClassRequest](
```

Comment:
Could we instead of using an additional command have a `redirect` option as a result?

Reply:
Like we agreed, we could do that later. I'll add some comment to the additional command that it'll be removed in the future.